File: javaClient/model/src/main/java/ro/transportcompany/Office.java
package ro.transportcompany;

public class Office extends Entity<Integer> {
    private String name;

    public Office() {
        super();
    }

    public Office(Integer id, String name) {
        super(id);
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "Id=" + getId() + " " + name;
    }
}

File: csharpServer/model/Office.cs
namespace model;

public class Office : Entity<int>
{
    public string Name { get; set; }

    public Office() { }

    public Office(int id, string name) : base(id)
    {
        Name = name;
    }

    public override string ToString()
    {
        return $"Office {{ Id = {Id}, Name = {Name} }}";
    }
}